src/eval-frame/components/panes/report-pane.js

```diff
@@ -9,29 +9,46 @@ import MarkdownItAnchor from 'markdown-it-anchor'
 const mdIt = MarkdownIt({ html: true })
 mdIt.use(MarkdownItKatex).use(MarkdownItAnchor)
 
+const mdDiv = html => (<div
+  className="user-markdown"
+  dangerouslySetInnerHTML={{ __html: html }} // eslint-disable-line react/no-danger
```

Comment:
Is there any way we can avoid using dangerouslySetInnerHTML here? It doesn't *seem* like it should be necessary.

Reply:
unfortunately, i don't think there is a better way to do this. The markdown-it parser eats a markdown string and returns an HTML string. I've looked at a couple markdown-it react wrappers, but they just use dangerouslySetInnerHTML under the hood.